smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy

```diff
@@ -120,7 +146,65 @@ use smithy.test#httpResponseTests
             doubleValue: "-Infinity",
             floatValue: "-Infinity"
         }
-    }
+    },
+    {
+        id: "RpcV2CborSupportsIndefiniteLengthStrings",
```

Comment:
This test doesn't really make sense, how is the implementation supposed to know to encode this way?

Reply:
I've added the `appliesTo: "server"` to this one. The server should be capable of deserializing indefinite length text strings.